great-idea-website/index.html

```diff
@@ -15,49 +15,95 @@
 </head>
 
   <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
+   
+    <header>
+      <nav class="nav-bar">
+          <a href= "#">Services</a>
+          <a href= "#">Product</a>
+          <a href= "#">Vision</a>
+          <a href= "#">Features</a>
+          <a href= "#">About</a>
+          <a href= "#">Contact</a>
+          </nav>
+          
+          <img src= "img/logo.png" alt="Great Idea! Company logo.">
+        
+        </header> 
+        <article>
+          <h1 class="intro-title"> 
+          INNOVATION </br>
+             ON </br>
+             DEMAND</br>
+         <button>Get Started</button>
+         </h1>
+  </div>
+  <div class="header-right">
+    <img class="header-right-image">
+    <img src="img/header-img.png" alt="Image of a code snippet">
+    </article>
+    
+    <section>
+      <div>
+    <h3 class="Features">
+      Features
+  </h3>
+  <p class="features-paragraph">
   Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  About
+</p>
+</div>
+<div>
+<h3 class="about">
+    about
+  </h3>
+<p class="about-paragraph">
+Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+</p>
+</div>
+</section>
+
+
+<img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+
+<section>
+  <div>
+<h3 class="services">
+  Services</h3>
+  <p class="services-paragraph">
   Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
```

Comment:
For a better semantics, this div should be an `article`, kindly read more on article tag and how you can make the best use of it.